src/hotspot/cpu/aarch64/c1_MacroAssembler_aarch64.hpp

```diff
@@ -100,7 +100,7 @@ using MacroAssembler::null_check;
   // header_size: size of object header in words
```

Comment:
@rkennke Should this be updated to `base_offset_in_bytes` too?

Reply:
And description updated?

Reply:
Same of x86

Reply:
Hi Galder, unfortunately this PR has already been intergrated before your review comments. I've opened [JDK-8327361](https://bugs.openjdk.org/browse/JDK-8327361) to track the additional fixes.

Reply:
@galderz please review #18120, thank you!